javascript/index.js

```diff
@@ -13,16 +13,27 @@ let milDec = document.getElementById('milDec');
 let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
-function printTime() {
-  // ... your code goes here
+function printTime(minutes, seconds) {
+  printMinutes(minutes)
+  printSeconds(seconds)
+
+
 }
 
-function printMinutes() {
-  // ... your code goes here
+function printMinutes(minutes) {
+ chronometer.twoDigitsNumber(chronometer.getMinutes(minutes))
```

Comment:
getMinutes should not have any parameters as well. You can see that inside the chronometer it works with the currentTime variable. So it doesn't need any variable passed via parameter.